static/css/components/generic-dropper.less

```diff
@@ -0,0 +1,86 @@
+.dropper-wrapper {
+  position: relative;
+
+  z-index: @z-index-level-4;
+
+  &--active {
+    z-index: @z-index-level-5;
+  }
+}
+
+.generic-dropper {
+  box-shadow: 0 0 0 1px @mid-grey;
+  border-radius: 4px;
+  position: absolute;
+  width: 100%;
+  z-index: @z-index-level-14;
+
+  &__actions {
+    display: flex;
+    flex-direction: row;
+    align-items: stretch;
+    cursor: pointer;
+
+    * {
+      color: @dark-grey;
+      background-color: @lightest-grey;
+    }
+
+    .generic-dropper__primary {
+      flex: 1;
+      margin: auto;
+
+      /* stylelint-disable max-nesting-depth */
+      button {
+        border: 2px solid @grey-f3f3f3;
+        border-right: 1px solid @lighter-grey;
+        border-bottom-right-radius: 0;
+        border-top-right-radius: 0;
+        cursor: pointer;
+        font-size: 1em;
+        width: 100%;
+        padding: 7px 0;
+        margin: 0;
+      }
+      /* stylelint-enable max-nesting-depth */
+    }
+    .generic-dropper__dropclick {
+      border-radius: 4px;
+      border-bottom-left-radius: 0;
+      border-top-left-radius: 0;
+      .display-flex();
+
+      /* stylelint-disable max-nesting-depth */
+      h3 {
+        flex: 1;
+      }
+
+      .arrow {
+        width: 23px;
+        margin: 6px 5px;
+        background-image: url(/images/icons/icon_dropit.png);
+        background-repeat: no-repeat;
+        filter: grayscale(100%);
+        background-position: 0 0;
+        background-size: 45px;
+
+        /* stylelint-disable selector-max-specificity */
+        &.up {
+          background-position: -22px 0;
+        }
+        /* stylelint-enable max-nesting-depth */
+        /* stylelint-enable selector-max-specificity */
+      }
+    }
+  }
+  &__dropdown {
+    display: none;
+    width: 100%;
+    // border: 1px solid @mid-grey;
+    border-top: 1px solid @lighter-grey;
+    position: absolute;
+    // z-index: @z-index-level-15;
+  }
+}
+
+@import (less) "components/mybooks-dropper.less";
```

Comment:
This seems backwards.  `mybooks-dropper.less`  should import the generic dropper styles.